### ISSUE TITLE
DST bug

### ISSUE THREAD

**Issue by lklepner** [kelektiv/node-cron#120]
Hi,

It seems I've encountered a bug related to DST changes. I'm running a function every minute using the following code -

```
var cronJob = require('cron').CronJob;
var job = new cronJob({
  cronTime: '00 */1 * * * *',
  onTick: function() {
    beginScheduledUpdate()
  },
  start: false,
  timeZone: "America/New_York"
});
job.start();
```

Last night when the DST change occurred my log showed it running normally until right after the change occurred. 

The last two lines in my log file read -
Beginning update: 2014-11-02 01:59
Beginning update: 2014-11-02 01:00

I would have expected the process to continue running once per minute but seems to have stopped at this point. I will conduct more extensive testing in the coming week to see if I can reproduce this error but I figured I should write in should this be a known issue. 

Thanks,
Lou

**Comment by coldlamper** [comment]
I experienced this same issue on March 8 at 2am.  I was running node-cron 1.0.5.  I could reproduce it by setting the date to 1:58am and letting it tick to 2:00am.
Updating to 1.0.9 seems to fix the issue.

**Comment by ncb000gt** [comment]
Well, that's good news. :) Thanks. I'll keep this open a little longer to field any others chiming in on it. But I'll close shortly if nobody else is having issues.

**Comment by sJhonny-e** [comment]
@ncb000gt we experienced a different problem with DST at the latest time change (march 8th in the US)- 
several jobs, which was scheduled at  0 00 07 * * ? (every day at 7AM) (some were scheduled for other times of day, but always once a day), were triggered at 1:00:00AM, 1:00:44AM, multiple times at 1:03:00AM (which is 1 hour before the clock changed), and many more times.

could this be related to the problem described here? 
I tried to reproduce this but no luck so far.
we're using node 0.10.33, node-cron 1.0.1 on a windows machine.
